sdk/communication/Azure.Communication.CallingServer/src/Azure.Communication.CallingServer.csproj

```diff
@@ -1,13 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>
-      This client library enables working with the Microsoft Azure Communication Service Calling Server APIs.
+      This client library enables working with the Microsoft Azure Communication CallingServer service.
```

Comment:
Can we rename to Calling Server, two separate words?

Reply:
We prefer using "CallingServer" which was "Calling Server" to keep it consistent within SDK library and API